### PR TITLE
Add ETX status functionality

### PR DESCRIPTION
**Pull request by robschleusner**
Fixes bug where ETXs are classified as contract interactions
Marks ETXs as External Send / Receive

### CODE REVIEW

**Review by Denis2626 — APPROVED** [review]

File: background/main.ts [hunk 1]
```diff
@@ -869,29 +866,75 @@ export default class Main extends BaseService<never> {
   }
 
   async enrichActivities(addressNetwork: AddressOnNetwork): Promise<void> {
-    const accountsToTrack = await this.chainService.getAccountsToTrack()
```
Comment (jdowning100) [inline background/main.ts:872]: Why did you remove this?

File: background/redux-slices/selectors/activitiesSelectors.ts [hunk 2]
```diff
@@ -7,6 +7,7 @@ export const selectCurrentAccountActivities = createSelector(
   selectCurrentAccount,
   selectCurrentNetwork,
   (activities, account, network) => {
+    console.log("All activites: ", activities)
```
Comment (jdowning100) [inline background/redux-slices/selectors/activitiesSelectors.ts]: Do we need this?

File: background/redux-slices/selectors/activitiesSelectors.ts [hunk 3]
```diff
@@ -17,7 +18,8 @@ export const selectActivitesHashesForEnrichment = createSelector(
     // Only need to enrich current activities if they don't have a blockHash already
     // Once they have a blockhash, they've been included and enriched already
     return currentActivities.flatMap((activity) =>
-      activity.blockHash == undefined ? activity.hash : []
+      //activity.status !== -100 ? {hash: activity.hash, status: activity.status, to: activity.to, from: activity.from} : []
```
Comment (jdowning100) [inline background/redux-slices/selectors/activitiesSelectors.ts]: Please remove this comment

File: background/services/chain/index.ts [hunk 4]
```diff
@@ -1907,6 +2036,31 @@ export default class ChainService extends BaseService<Events> {
     this.queueTransactionHashToRetrieve(network, transaction.hash, Date.now())
   }
 
+  private async subscribeToETXConfirmation(
+    network: EVMNetwork,
+    itx: AnyEVMTransaction,
+    etx: AnyEVMTransaction
+  ): Promise<void> {
+    console.log("ROB LOG - Subscribing to ETX Confirm")
```
Comment (jdowning100) [inline background/services/chain/index.ts]: Is this still necessary?

**Review by Denis2626 — DISMISSED** [review]

File: background/main.ts (hunk 1 above)
Comment (robschleusner) [reply background/main.ts:872]: I didn't remove it. I just moved it into each individual "enrichITXActivity" and "enrichETXActivity" functions, since it wasn't needed in overall "enrichActivities"

File: background/redux-slices/selectors/activitiesSelectors.ts (hunk 2 above)
Comment (robschleusner) [reply background/redux-slices/selectors/activitiesSelectors.ts]: No

**Review by jdowning100 — APPROVED** [review]

**Review by wizeguyy — APPROVED** [review]
